ext/phar/phar_object.c

```diff
@@ -1809,7 +1813,9 @@ PHP_METHOD(Phar, buildFromIterator)
 
 	if (SUCCESS == spl_iterator_apply(obj, (spl_iterator_apply_func_t) phar_build, (void *) &pass)) {
 		phar_obj->archive->ufp = pass.fp;
-		phar_flush(phar_obj->archive, &error);
+		// TODO: Set to &pass; struct?
```

Comment:
If buildFromIterator() would have a parameter to specify the timestamp, this would be so golden. This is a longtime issue  for me when I archive inflight (build) trees. @Girgias Thanks a lot for all already, even if not yet that. /cc @Seldaek FYI.